Object.h

```diff
@@ -10,6 +10,7 @@ class Object {
 
 private:
 	int x, y, width, height;

```

Comment:
So if you wanted to make a field for SDL_Point here, you can put all the get_position/set_position stuff here.